source/common/stream_info/stream_info_impl.h

```diff
@@ -322,6 +334,8 @@ struct StreamInfoImpl : public StreamInfo {
 
   uint64_t bytes_received_{};
   uint64_t bytes_sent_{};
+  uint64_t wire_bytes_received_{};
```

Comment:
Any reason we just can't have the bytes sent / received tracked in the stream_info impl vs in both stream_info_impl and stream? Why do we need it in both places?

Reply:
Technically we don't need it in both places, will refactor.